src/instrumentation/historian.ts

```diff
@@ -62,6 +69,28 @@ export default class Historian {
         }
     }
 
+    ignoreUncaughtExceptions() {
+        let p;
+        try {
+            // Use eval to hide process usage from Webpack and Browserify.
+            p = eval('process');
+        } catch {}
+        if (typeof p === 'object' && typeof p.on === 'function') {
+          p.removeListener('uncaughtException', this.uncaughtExceptionHandler);
+        }
+    }
+
+    ignoreUnhandledRejections() {
```

Comment:
why not just call out to ignoreUncaughtExceptions for now? small pref for merging shared code, nitpick

Reply:
talked about